HW5/functional/app.js

```diff
@@ -0,0 +1,54 @@
+import { modal } from "./modal.js";
+import { showModal, hideModal } from "./modalActions.js";
+import { todoItem, renderTaskList } from "./todoItem.js";
+import { search } from "./search.js";
+console.log("app runs");
+
+export function tasksRender() {
+  const body = document.querySelector("body");
+
+  const main = document.createElement("section");
+  main.setAttribute("class", "main__container");
+  body.appendChild(main);
+  modal();
```

Comment:
With this approach, when you reload the page (for example, when you select the goal as completed) this modal is appearing for a moment

Reply:
yes , I noteced it, and it was changed in hw6 PR already, modal is created and shown only when "add Task" is clicked.
  newTaskButton.addEventListener("click", () => {
    modal();
    showModal();
  });